### PR TITLE
fix(Clowder): RHCLOUD-20284 update navItems in deploy template

### PR DESCRIPTION
**Pull request by BlakeHolifield**
updates the `navItems` field in the CRD to match https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/rhel-navigation.json#L102 

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [X] General Coding Practices

### CODE REVIEW

File: deployment_template.yaml [hunk 1]
```diff
@@ -19,10 +19,21 @@ objects:
...
       navItems:
-        - appId: "compliance"
-          title: "Compliance"
-          href: "/insights/compliance"
-          product: "Red Hat Insights"
+        - expandable: true
+          title: Compliance
+          routes:
+          - appId: compliance
+            href: /insights/compliance/report
```
Comment (LightOfHeaven1994) [inline deployment_template.yaml:26]: typo: should be `reports`

**Comment by BlakeHolifield** [comment]
Did a linting dependency update recently? I'm seeing 
```
> compliance-frontend@1.1.0 lint:commit

> commitlint --from HEAD~$(git rev-list --count HEAD ^master) --to HEAD --verbose
```
In both Travis and Jenkins builds.

**Review by skateman — APPROVED** [review]

**Pull request merged**